media-gfx/sane_backends/sane_backends-1.3.1.recipe

```diff
@@ -78,20 +78,20 @@ BUILD_PREREQUIRES="
 	cmd:autoconf
 	cmd:automake
 	cmd:autoreconf
-	cmd:autopoint
+	cmd:autopoint$secondaryArchSuffix
```

Comment:
Make sure to use autopoint from gettext_x86 (this "can" cause weird behaviours with the counterpart).